server/controllers/auth/login.js

```diff
@@ -7,16 +8,16 @@ const login = async (req, res, next) => {
   const { createSession } = require('./index'); // there is a problem with requiring this function
   try {
     const { password, email } = req.body;
-    await loginSchema.validateAsync(req.body).catch((err) => {
-      res.status(422).json({ message: err.message });
+    await loginSchema.validateAsync(req.body).catch((error) => {
+      throw { message: error.details[0].message, status: 400 };
```

Comment:
why you do that?

Reply:
to send the error in the catch block with its status and next to the error handler